sale_order_import/wizards/sale_order_import.py

```diff
@@ -144,13 +144,14 @@ def _get_order_value(self, error_vals, taxes, taxes_id, price_unit, product_qty)
             self._get_taxes(tax_from_chunk, taxes, error_vals)
         price_unit = float(price_unit)
         product_qty = float(product_qty)
-        for k, v in {_("Quantity"): product_qty, _("Unit Price"): price_unit}.items():
-            if v >= 0:
-                continue
-            error_vals["error_message"] = (
-                error_vals["error_message"]
-                + _("Value must be 0 or greater: %s") % k
-                + "\n"
+        if price_unit < 0:
+            error_vals["error_message"] += (
+                _("Value must be 0 or greater: %s") % _("Unit Price") + "\n"
```

Comment:
```suggestion
                _("Value must be 0 or greater: Unit Price") + "\n"
```
There is no point in using a placeholder now.